com.note.slayer.yaml

```
@@ -0,0 +1,41 @@
+app-id: com.note-slayer.app
```

```suggestion
app-id: com.note_slayer.app
```

https://docs.flatpak.org/en/latest/conventions.html#application-ids

---

but `app` as the application name part of the ID is ambiguous. This should be something clear like "NoteSlayer".